var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -140,7 +142,7 @@ class Opencv(CMakePackage, CudaPackage):
     patch("opencv3.4.12_clp_cmake.patch", when="@3.4.12")
     patch("opencv3.3_clp_cmake.patch", when="@:3.4.6")
 
-    patch("opencv3.4.4_cvv_cmake.patch", when="@3.4.4:")
+    patch("opencv3.4.4_cvv_cmake.patch", when="@3.4.4:4.5")
```

Comment:
It looks like this patch has already been applied to the contrib source code, it already uses `HAVE_QT` instead of `HAVE_QT5`.